megamek/src/megamek/server/Server.java

```diff
@@ -10802,8 +10802,10 @@ public void deliverMortarFlare(Coords coords, int duration) {
      */
 
     public void deliverMissileSmoke(Coords coords, int smokeType, Vector<Report> vPhaseReport) {
-        Report r = new Report(5185, Report.PUBLIC);
+        Report r = new Report(5183, Report.PUBLIC);
         r.indent(2);
+        //Report either light or heavy smoke, as appropriate
+        r.choose(smokeType == 1);
         r.add(coords.getBoardNum());
         vPhaseReport.add(r);
         createSmoke(coords, smokeType, 3);
```

Comment:
Per TacOps, p. 371:
>>>The Smoke generated by Smoke LRMs will linger for a number of turns equal to the half the rack size of the launcher used (rounded down) before dissipating in the End Phase of the final turn."

You'll need to overload this method to allow you to pass the duration as a parameter.

Reply:
Errata v3.4 has changed this to a flat 3 rounds.

Reply:
I'm looking at the 2018 printing that is supposed to incorporate all the errata. Is there new errata since then or did this one not get incorporated?

Reply:
Looking at previous editions, 2008 has salvo size/2, 2012 has a flat three rounds, and 2018 has salvo size/2 again. It looks like it was changed but the 2018 printing has the older rule in error.